modules/gis/test/reproject.spec.js

```diff
@@ -0,0 +1,62 @@
+import test from 'tape-promise/tape';
+import {mapBinaryCoords, mapGeoJsonCoords} from '@loaders.gl/gis';
+import {Proj4Projection} from '@math.gl/proj4';
```

Comment:
Since I ended up making an arbitrary `mapCoords` wrapper and since Proj4Projection is tested within math.gl already, there's no need to test reprojections in this test file. I could just test `mapCoords(features, coord => [coord[0] + 1, coord[1] + 2]`.

Could make for simpler tests.

Reply:
Sure. The key is to test that coordinates are updated. But doesn't hurt to make sure projections work.